### PR TITLE
fix: Change rootfs compression to gzip

### PR DESCRIPTION
**Pull request by vsiravar**
Issue #, if available:

*Description of changes:*
Change compression of rootfs bundle to gzip since windows does not come with zstd installed. 

*Testing done:*
Yes, locally. 

- [X] I've reviewed the guidance in CONTRIBUTING.md

#### License Acceptance

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by pendo324 — APPROVED** [review]

File: .github/workflows/cdk-e2e.yaml
Comment (pendo324) [inline .github/workflows/cdk-e2e.yaml:1]: Accidentally pushed to wrong branch?

Comment (vsiravar) [reply .github/workflows/cdk-e2e.yaml:1]: :( my bad. My fork was out of sync with source. Updated.

Comment (pendo324) [reply .github/workflows/cdk-e2e.yaml:1]: No worries, lgtm now

**Review by pendo324 — APPROVED** [review]

**Comment by austinvazquez** [comment]
Do we need to update `Makefile` and `bin/update-rootfs.sh` now as well? See https://github.com/search?q=org%3Arunfinch%20.tar.zst&type=code

**Comment by vsiravar** [comment]
> Do we need to update `Makefile` and `bin/update-rootfs.sh` now as well? See https://github.com/search?q=org%3Arunfinch%20.tar.zst&type=code

This is a good point. However the Makefile is updated when https://github.com/runfinch/finch-core/blob/main/.github/workflows/update-dependencies.yaml#L32 action runs. For the next run we still need https://github.com/runfinch/finch-core/blob/main/bin/update-rootfs.sh#L5 as the Makefile has a zst compressed tar. Once the change is merged, I will update the `update-rootfs.sh` to 
```
AARCH64_FILENAME_PATTERN="common/aarch64/finch-rootfs-production-arm64-[0-9].*\.tar.gz$"
AMD64_FILENAME_PATTERN="common/x86-64/finch-rootfs-production-amd64-[0-9].*\.tar.gz$"
```

**Review by austinvazquez — APPROVED** [review]

**Pull request merged**